### PR TITLE
Add Install NVIDIA driver

### PR DESCRIPTION
**Pull request by DanielHo-BS**
Add Install NVIDIA driver:
  1. roscube-i/images
  2. roscube-i/index.rst
  3. roscube-i/nvidia_driver.rst

### CODE REVIEW

File: roscube-i/nvidia_driver.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
+.. _nvidia_driver:
+
+How to Install NVDIA Driver
+###########################
+
+The tutorial will guide you how to install nvidia driver.
+
+Supported Model: ROScube-I series
```
Comment (evshary) [inline roscube-i/nvidia_driver.rst]: ROScube-I series => ROScube-I-E
Only the extension box version can go with GPU.

File: roscube-i/nvidia_driver.rst [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+Here is what we see:
+
+.. image:: images/ubuntu-driver-devices.png
+  :width: 80%
+  :align: center
+
+3. Install the GPU driver. 
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Here show two way how to install driver:
```
Comment (evshary) [inline roscube-i/nvidia_driver.rst]: Here show two ways how to install the driver:

File: roscube-i/nvidia_driver.rst [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    * Automatically installtion
```
Comment (evshary) [inline roscube-i/nvidia_driver.rst]: installtion => installation

File: roscube-i/nvidia_driver.rst [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    * Custom installation
+
+Automatically installtion
```
Comment (evshary) [inline roscube-i/nvidia_driver.rst]: installtion => installation

File: roscube-i/nvidia_driver.rst [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+-------------------------
+
+Let us install recommended driver automatically:
+
+.. code-block::
```
Comment (evshary) [inline roscube-i/nvidia_driver.rst]: Wrong code-block syntax.
`.. code-block:: bash`

File: roscube-i/nvidia_driver.rst [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block::
+
+    sudo reboot
+
+5. Verification.
+^^^^^^^^^^^^^^^^
+
+Open the terminal application and type ``nvidia-smi`` to see GPU info and process that are using Nvidia GPU:
```
Comment (evshary) [inline roscube-i/nvidia_driver.rst]: Nvidia=>NVIDIA

**Review by evshary — CHANGES_REQUESTED** [review]

File: roscube-i/nvidia_driver.rst (hunk 1 above)
Comment (DanielHo-BS) [reply roscube-i/nvidia_driver.rst]: Done

File: roscube-i/nvidia_driver.rst (hunk 2 above)
Comment (DanielHo-BS) [reply roscube-i/nvidia_driver.rst]: Done

File: roscube-i/nvidia_driver.rst (hunk 3 above)
Comment (DanielHo-BS) [reply roscube-i/nvidia_driver.rst]: Done

File: roscube-i/nvidia_driver.rst (hunk 4 above)
Comment (DanielHo-BS) [reply roscube-i/nvidia_driver.rst]: Done

File: roscube-i/nvidia_driver.rst (hunk 5 above)
Comment (DanielHo-BS) [reply roscube-i/nvidia_driver.rst]: Done

File: roscube-i/nvidia_driver.rst (hunk 6 above)
Comment (DanielHo-BS) [reply roscube-i/nvidia_driver.rst]: Done

**Review by DanielHo-BS — COMMENTED** [review]
Done the Request changes

Add the GPU card
  1. roscube-i/gpu_card/images
  2. roscube-i/gpu_card/index.rsr
  3. roscube-i/gpu_card/nvidia_driver.rst
  4. roscube-i/gpu_card/supported_gpu_model

**Comment by DanielHo-BS** [comment]
1. Add "GPU performance" to ROScube-X/advanced and ROScube-pico/advanced.
2. Update "GPU card" in ROScube-I.
3. Fix some English grammar.

**Comment by evshary** [comment]
LGTM

**Pull request merged**